public/workspace-encounters.json

```diff
@@ -8,44 +8,33 @@
       ],
       "zoom": 3,
       "layers": [
-        {
-          "title": "AIS Fishing effort",
-          "description": "AIS Fishing effort",
-          "url": "https://api-dot-world-fishing-827.appspot.com/v2/tilesets/516-resample-v2",
-          "visible": true,
-          "hue": 182,
-          "type": "ClusterAnimation",
-          "opacity": 1,
-          "id": "fishing2",
-          "tilesetId": "516-resample-v2"
-        },
-        {
-         "title": "VMS Fishing effort",
-         "description": "VMS Fishing effort",
-         "url": "https://api-dot-world-fishing-827.appspot.com/v2/tilesets/525-indo-public-parametrize-v4",
-         "visible": true,
-         "hue": 60,
-         "type": "ClusterAnimation",
-         "opacity": 1,
-         "id": "fishing",
-         "tilesetId": "525-indo-public-parametrize-v6"
-        },
         {
           "title": "Encounters",
           "description": "Encounters",
-          "url": "https://api-dot-world-fishing-827.appspot.com/v2/tilesets/vizz-generated-encounters-pbf-v2",
+          "url": "https://api-dot-skytruth-pleuston.appspot.com/v2/tilesets/encounters-layer-pbf-v5",
           "visible": true,
           "hue": 0,
           "type": "Encounters",
           "opacity": 1,
           "id": "encounters",
-          "tilesetId": "vizz-generated-encounters-pbf-v2"
+          "tilesetId": "vizz-generated-encounters-pbf-v5"
+        },
+        {
+          "title": "AIS Fishing effort",
+          "visible": true,
+          "description": "AIS Fishing effort",
+          "url": "https://api-dot-skytruth-pleuston.appspot.com/v2/tilesets/gfw-tasks-657-uvi-v1",
```

Comment:
@nerik we should avoid having workspaces pointing to our dev environment on the repository. We need to plan moving away from this. We can have a separate repo with resources for us to use that has this workspace.  Leave this for now. I'll create a ticket for later.

Reply:
@enriquetuya I had git <a href="https://github.com/GlobalFishingWatch/map-client/blob/develop/.gitignore#L24">ignoring</a> that workspace - as its only use is for local dev. I also removed the reference to a dev env tileset that was in the default local workspace (workspace.json)